include/communication_interface.h

```diff
@@ -125,9 +131,13 @@ class CommunicationInterface {
 
   std::string lcm_driver_status_channel_;
   std::string lcm_pause_status_channel_;
-  std::string lcm_user_stop_channel_;
   std::string lcm_brakes_locked_channel_;
 
+  std::string lcm_user_stop_channel_;
+  std::string sim_u_stop_source;
+  bool sim_u_stop {false};
+  bool prev_pause_state {false};
```

Comment:
Same as above; similar to https://github.com/DexaiRobotics/ancillary_arm/pull/77 I don't think you need `prev_pause_state`